luci_grpc_interface/client/src/client.cpp

```diff
@@ -291,13 +375,14 @@ void ClientGuide::readJoystickScalingData() const
     JoystickData response;
 
     std::unique_ptr<ClientReader<JoystickData>> reader(
-        stub_->ScaledJoystickStream(&context, request));
+        stub_->DriveJoystickStream(&context, request));
```

Comment:
We don't really have an exact stream that has this data currently. The closest we have is the `ActiveScalingStream` that has the currently used scaling values in it. I believe this `ScaledJoystickStream` was being used to back out the current scaling values since it was hard for us to provide before, so I think replacing it with using the `ActiveScalingStream` is probably the right move. I'm pretty sure we don't have a `ScaledJoystickStream` anymore because it was only for that reason, but we could add it back if it's useful